app/models/project.js

```diff
@@ -77,7 +75,9 @@ export default class ProjectModel extends Model {
   @attr() bbls;
 
   @attr({ defaultValue: () => EmptyFeatureCollection })
-  bbl_featurecollection;
+  bbl_featurecollection
+
+  @attr() milestones;
```

Comment:
Why going back to POJO milestones? 
oh --- I'm guessing we're leaving that to a later commit?